### PR TITLE
Feature/sanity about page

### PR DESCRIPTION
**Pull request by kcbreuil**

### CODE REVIEW

File: studio/schemas/documents/historyYear.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        {
+            name: 'historyText',
+            title: 'History Text',
+            type: 'text',
+        },
+        {
+            name: 'image',
+            title: 'Image',
+            type: 'image',
+        },
```
Comment (asasmith) [inline studio/schemas/documents/historyYear.js:20]: Lets create a custom image object for all images that aren't background images (similar to [this](https://github.com/bckrm/gianna-biscontini/blob/main/studio/schemas/objects/mainImage.js))

File: studio/schemas/documents/leaderCard.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            type: 'string',
+        },
+        {
+            name: 'image',
+            title: 'Image',
+            type: 'image',
+            options: {
+                hotspot: true,
+            },
+        },
```
Comment (asasmith) [inline studio/schemas/documents/leaderCard.js:23]: Same comment as above

**Review by asasmith — CHANGES_REQUESTED** [review]

File: studio/schemas/documents/historyYear.js (hunk 1 above)
Comment (kcbreuil) [reply studio/schemas/documents/historyYear.js:20]: @asasmith sure thing: I was looking in the documentation last night about this. When do we use a custom image vs. the image type bootstrapped in sanity?

Comment (kcbreuil) [reply studio/schemas/documents/historyYear.js:20]: or a custom block type vs the block type?

File: studio/schemas/documents/leaderCard.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+            title: 'Title',
+            type: 'string',
+        },
+        {
+            name: 'image',
+            title: 'Image',
+            type: 'mainImage',
+            options: {
+                hotspot: true,
+            },
```
Comment (asasmith) [inline studio/schemas/documents/leaderCard.js:22]: I don't think you need this if it's defined in the `mainImage` object

**Review by asasmith — CHANGES_REQUESTED** [review]

File: studio/schemas/documents/leaderCard.js (hunk 3 above)
Comment (kcbreuil) [reply studio/schemas/documents/leaderCard.js]: @asasmith updated

File: studio/schemas/documents/aboutSection.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    name: 'aboutSection',
+    title: 'About Section',
+    type: 'document',
+    fields: [
+        {
+            name: 'leftAboutText',
+            title: 'Left About Text',
+            type: 'array',
+            of: [{ type: 'block' }],
+        },
```
Comment (asasmith) [inline studio/schemas/documents/aboutSection.js:11]: should probably be `leftColumn`/`rightColumn`. I think text is implied so I don't think it's necessary in the name.

File: studio/schemas/documents/aboutSection.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+        {
+            name: 'aboutIconText',
+            title: 'About Icon Text',
+            type: 'array',
+            of: [{ type: 'reference', to: { type: 'aboutIcon' } }],
+        },
```
Comment (asasmith) [inline studio/schemas/documents/aboutSection.js:17]: this should be included in the body text for each of these columns, not it's own separate field.

**Review by asasmith — CHANGES_REQUESTED** [review]

**Review by asasmith — APPROVED** [review]

**Pull request merged**